source/_integrations/agent_dvr.markdown

```diff
@@ -5,7 +5,7 @@ ha_category:
   - Alarm
   - Camera
 ha_config_flow: true
-ha_release: '0.110'
+ha_release: 0.110
```

Comment:
These are strings on purpose. In numbers it will be interpreted as `0.11`

Reply:
Ah, that would indeed be not the preferred result. Any suggestion to get this working with the url? can I replace the quotes?